helpdesk/views/staff.py

```diff
@@ -409,7 +409,7 @@ def return_ticketccstring_and_show_subscribe(user, ticket):
     counter_all_ticketcc = len(all_ticketcc) - 1
     show_subscribe = True
     for i, ticketcc in enumerate(all_ticketcc):
-        ticketcc_this_entry = str(ticketcc.display)
+        ticketcc_this_entry = str(ticketcc.display).strip()
```

Comment:
Why is this needed and why should the strip be here and not here: https://github.com/django-helpdesk/django-helpdesk/blob/012ba4f42d55282e81419735f15fd81fd44f50b8/helpdesk/models.py#L1708

Reply:
agree